CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## Unreleased
+- Feat: Add support for key commands (backspace and esc)
```

Comment:
markdownlint should complain about this if installed to VS Code (we dont have precommit hook or any config for that yet), but the heading should have an empty line before the list (as shown in the other sections)

Reply:
I guess I should install markdownlint then :) I assumed everything related to linting has been configured to rpe-commit and thus using other linters on top of that would only make a mess.

I will fix this.